GithubAPIPractice/ViewModels/IssueListViewModel.swift

```diff
@@ -29,13 +29,20 @@ final class IssueListViewModel: IssueListViewModelOutput {
     //(refs: "https://stackoverflow.com/questions/54213183/display-activity-indicator-with-rxswift-and-mvvm")
     
 //    privateをつけないと、ViewController側からViewModel内のshowLoadingに対して直接acceptしてデータを流すことが出来てしまいます。
-    
-
+    private let showLoadingRelay = BehaviorRelay<Bool>(value: true)
+    var showLoading: Driver<Bool> {
+        showLoadingRelay.asDriver()
+    }
 
-    let showLoading = BehaviorRelay<Bool>(value: true)
     
     /*Outputに関する記述*/
+    //ここもDriverに変換できる？
     lazy var dataRelay = BehaviorRelay<[SectionModel]>(value: [])
+    //↓だとうまくいかなかった
```

Comment:
ここはどう上手くいかなかったのかな👀

Reply:
ViewController側でdriveしたらどうなったか教えてしいです

Reply:
SignalかObservableを使うのも検討できそうですが、Driver使おうとしている理由ってどのあたりかな？

Reply:
なんとなくでDriverを選んでしまっていたので、もう少しやりたいことと照らし合わせてストリームを選べるように気をつけます。
FBで頂いていた、各種View用の出力情報をDriverとして出力することができると尚良い
を意識してDriverを選んだ気がしますが、sectionModelの取得を行う際はDriverはあまり適切ではないような気がしたので、SignalかObservableでの実装を考えてみようと思います。
うまくいかなかったのは、
上記のコードだと
Property 'dataRelay' must be declared internal because it matches a requirement in internal protocol 'IssueListViewModelOutput'
と出てきており、Outputで定義した形と異なるためエラーが出てしまいました。

Reply:
おっけいです。
SignalとDriverはUI処理用に用意されてるので、このレイヤーだとこのどちらかを使うことが多いです。

考えまとまったらコメントも添えて組んでみて👍